LoadingContainer.js

```diff
@@ -58,6 +58,11 @@ export default class LoadingContainer extends React.Component {
     this._loadingOverlay.showOverlay();
   }
 
+  async reloadAsync() {
+    this.setState({ isLoading: false });
```

Comment:
probably need to think more about concurrency here -- if this container is already in the middle of loading data, it doesn't get canceled and it will call onReadyAsync when it's complete.